gnovm/cmd/gno/bug.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+func newBugCmd(io *commands.IO) *commands.Command {
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "bug",
+			ShortUsage: "bug",
+			ShortHelp:  "start a bug report",
+		},
+		commands.NewEmptyConfig(),
+		func(_ context.Context, args []string) error {
+			return execBug(args, io)
+		},
+	)
+}
+
+func execBug(args []string, io *commands.IO) error {
+	if len(args) != 0 {
+		return flag.ErrHelp
+	}
+
+	var buf strings.Builder
+	buf.WriteString(bugHeader)
+	writeEnvironment(&buf)
+	buf.WriteString(bugFooter)
+	// TODO: include gno version or commit?
+
+	body := buf.String()
+	url := "https://github.com/gnolang/gno/issues/new?body=" + url.QueryEscape(body)
```

Comment:
what happens when body is too long?

Reply:
i don't think we need to worry about that here. Since `body` is not very dynamic. It will always be under 500chars. 
which is totally under the safe limit(2000chars?)?

Reply:
WDYT of this check? (if we are still worried about `body` being too long)
```go
if len(url) > 2048 {
    io.Println("Please file a new issue at github.com/gnolang/gno/issues/new using this template:")
    io.Println()
    io.Println(body)
}
```

Reply:
Done b5673a11ecabfcc83980eac791903eec3f947881 ✅